packages/debug/src/browser/editor/debug-hover-widget.ts

```diff
@@ -56,7 +56,8 @@ export function createDebugHoverWidgetContainer(parent: interfaces.Container, ed
 @injectable()
 export class DebugHoverWidget extends SourceTreeWidget implements monaco.editor.IContentWidget {
 
-    protected readonly toDispose = new DisposableCollection();
+    // TODO: Is this still needed?
+    // protected override toDispose = new DisposableCollection();
```

Comment:
Do you want to address this as part of this PR?

Reply:
Honestly if we catch bugs by turning this flag on then yes:

If we keep this line, it will override the previous declaration of the same field. Except that the previous declaration holds references to several disposables. Is that a bug?

Reply:
We have `DebugHoverWidget#toDispose > SourceTreeWidget > TreeWidget > ReactWidget > Widget#toDispose`.

And it looks like a mistake to override `toDispose` in `DebugHoverWidget`.

Reply:
I agree, it seems as though the `DisposableCollection` from the super class should not be clobbered here. The first bug fixed by the `override` marker already. :-)